store.go

```diff
@@ -1932,13 +1944,50 @@ func (s *store) Mount(id, mountLabel string) (string, error) {
 	if modified, err := rlstore.Modified(); modified || err != nil {
 		rlstore.Load()
 	}
-	if rlstore.Exists(id) {
-		return rlstore.Mount(id, mountLabel)
+	if !rlstore.Exists(id) {
+		return "", ErrLayerUnknown
 	}
-	return "", ErrLayerUnknown
+
+	rootUID, rootGID, err := idtools.GetRootUIDGID(s.uidMap, s.gidMap)
+	if err != nil {
+		return "", err
+	}
+
+	merged := ""
+	if rootUID != 0 {
+		merged = filepath.Join(s.RunRoot(), id, "merged")
```

Comment:
What's the rationale for naming this "merged"?  Only the overlay driver names its mountpoints that.

Reply:
I was just copying the names from overlay.  But I agree, it is better to be more neutral, renamed to "root"